README.md

```diff
@@ -14,9 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-<h1><img alt="tool icon" src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />&nbsp;&nbsp;DeepSparse</h1>
 
-<h3>Neural network inference engine that delivers GPU-class performance for sparsified models on CPUs</h3>
```

Comment:
Original software tag lines need to remain for now as there is a separate review process for these, with several cross-department staff involved. They are used in multiple places so we want to change them relatively at once.